program/find-sum-of-cubes-of-numbers/find_sum_of_cubes_of_numbers.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "fmt"
+
+func findArraySum(arr []int) int {
+	res := 0
+	for i := 0; i < len(arr); i++ {
+		res += arr[i] * 3
```

Comment:
I think you forgot to calculate cube here

Reply:
sorry, yeah! i'll update 😅